### PR TITLE
adding new recipe burger.txt

### PR DESCRIPTION
**Pull request by saisatwik99**
[burger.txt.txt](https://github.com/teluskotraining/tusk/files/4142166/burger.txt.txt)

### CODE REVIEW

**Comment by narendra345** [comment]
h

**Comment by IKiriZudoI** [comment]
Comment

File: pizza.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+Recipe from Navin Reddy
+
+
+1. Go to Dominos.com
+2. Select the pizza.
```
Comment (ghost) [inline pizza.txt:7]: Choose Chicken Pizza

File: pizza.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+
+Recipe from Navin Reddy
```
Comment (ghost) [inline pizza.txt:3]: Recipe from Rick Mondal

**Comment by melika1386** [comment]
****

**Pull request closed without merging**